### ISSUE TITLE
Option to keep the same position on refresh

### ISSUE THREAD

**Issue by zilions** [germanysbestkeptsecret/Wookmark-jQuery#219]
It would be great to have an option that allows all items to keep their position on refresh.

For example, if an item has a height of 200px, but then gets extra content pushed to it after page load to make it 500px, all the other elements after it get moved around and reordered into different rows/columns. This can confuse the user since they thought that an item was in one place, yet it has now been moved completely.

Would this be a difficult option to add?

**Comment by Sebobo** [comment]
But the others have to be moved around to make room for the bigger item.
Or do you mean, you just want the columns to be reorderd and not the rows?

**Comment by zilions** [comment]
Correct. So if the height changes on a particular item, just to push down the items below it instead of reordering all the other items into different columns and confusing the user.
